docs/source/teams/installation.rst

```diff
@@ -320,26 +320,14 @@ here is an example configuration:
 
 .. code-block:: json
 
-    [
-        {
-            "AllowedHeaders": [
-                "*"
-            ],
-            "AllowedMethods": [
-                "GET",
-                "HEAD",
-            ],
-            "AllowedOrigins": [
-                "https://fiftyone-teams-deployment.yourcompany.com"
-            ],
-            "ExposeHeaders": [
-                "x-amz-server-side-encryption",
-                "x-amz-request-id",
-                "x-amz-id-2"
-            ],
-            "MaxAgeSeconds": 3000
-        }
-    ]
+[
+    {
+        "origin": ["https://fiftyone-teams-deployment.yourcompany.com"],
+        "method": ["GET", "HEAD"],
```

Comment:
Out of curiosity, is the "HEAD" method here necessary? It looks like it's not in [this example](https://cloud.google.com/storage/docs/cors-configurations) and not in the GCS test bucket we have

Reply:
From some quick reading, the "HEAD" method is used for to obtain metadata about the resource, such as content type, content length, and modification date, without downloading the entire resource. And the general recommendation is to list both methods since it could help with performance optimizations and pre-flight requests. So I think we can leave it as is.

Reply:
Gotcha, makes sense :+1: